cypress/integration/e2e/components/index.mdx.spec.ts

```diff
@@ -0,0 +1,14 @@
+describe('Index.mdx component documentation page', () => {
+  beforeEach(() => {
+    cy.visit('/components/index.mdx');
+  });
+
+  it('should render the index.mdx component page correctly', () => {
+    cy.pageHeaderShouldBeVisible('Index Mdx');
+
+    cy.checkInPageNavigationLinks();
+    cy.checkPageAside();
+    cy.checkLivePreviews();
+    cy.checkChangelogRevealer();
```

Comment:
The fact that this function doesn't fail on the overview page is concerning... There's no changelog on this page.

Reply:
It actually doesn't pass. The test suite just doesn't run the rest of the commands if the before/beforeEach hook fails, which is intended behavior.

<img width="591" alt="Screen Shot 2021-11-16 at 12 17 44 PM" src="https://user-images.githubusercontent.com/18120906/142059342-c3f4e6c0-02ef-4b7f-939d-c49e1de6ea11.png">

I just fixed the visit command and the whole test is failing as expected. This was just generated by the script, and isn't actually the test for the overview page. going to delete it.